.github/workflows/pythonpackage.yml

```diff
@@ -3,7 +3,11 @@
 
 name: Python package
 
-on: [push]
+on:
+  push:
+    branches:
+      - master
+  pull_request:
```

Comment:
do you want to limit pr action runs to only `master` as well?

Reply:
We could. But if we did like a wip branch and let people PR into the wip it wouldn't run. I don't think this one needs to be as explicit.